File: ClimbingGymCompanionServiceLambda/src/main/java/com/nashss/se/ClimbingGymCompanionService/enums/climbStatus.java
package com.nashss.se.ClimbingGymCompanionService.enums;

public enum climbStatus {
    // first attempt completion start to finish
    COMPLETED_FLASHED,
    // completion start to finish
    COMPLETED_SENT,
    // completion start to finish but fell off wall in between
    COMPLETED_IN_STAGES,
    // attempting to complete but it is a work in progress
    PROJECTING,
    // not yet climbed but want to
    WANT_TO_CLIMB,
    // attempted but too difficult to complete
    TOO_DIFFICULT
}

File: web/src/enums/climbStatus.js
// climbStatus.js

const climbStatus = {
    // first attempt completion start to finish
    COMPLETED_FLASHED: 'COMPLETED FLASED',
    // completion start to finish
    COMPLETED_SENT: 'COMPLETED SENT',
    // completion start to finish but fell off wall in between
    COMPLETED_IN_STAGES: 'COMPLETED IN STAGES',
    // attempting to complete but it is a work in progress
    PROJECTING: 'PROJECTING',
    // not yet climbed but want to
    WANT_TO_CLIMB: 'WANT TO CLIMB',
    // attempted but too difficult to complete
    TOO_DIFFICULT: ' TOO DIFFICULT'
};

export default climbStatus;